dev-lang/vala/vala-0.36.11.recipe

```diff
@@ -28,26 +31,28 @@ PROVIDES="
 	cmd:valac_${portVersion%.*}$secondaryArchSuffix
 	cmd:vapicheck_${portVersion%.*}$secondaryArchSuffix
 	cmd:vapigen_${portVersion%.*}$secondaryArchSuffix
-	lib:libvala_${portVersion%.*}$secondaryArchSuffix
-	"
-PROVIDES_common="
-	vala${secondaryArchSuffix}_common = $portVersion
+	lib:libvala_${portVersion%.*}$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
 	glib2${secondaryArchSuffix} >= 2.32.0
-	vala${secondaryArchSuffix}_common == $portVersion
+	lib:libdbus_1$secondaryArchSuffix
 	lib:libffi$secondaryArchSuffix
 	lib:libiconv$secondaryArchSuffix
 	lib:libintl$secondaryArchSuffix
 	"
-REQUIRES_common="
```

Comment:
I'd recommend adding this (right after the `REQUIRES`):
```
REPLACES="
	vala${secondaryArchSuffix}_common
	"
```
This would be handy for users upgrading from `vala-0.36.4-1` (or `vala_x86-0.36.4-1`) to `vala-0.36.11-1` (or `vala_x86-0.36.11-1`) as it would instruct pkgman to delete `vala_common-0.36.4-1` (or `vala_x86_common-0.36.4-1`).